demos/interactive_face_detection_demo/cpp/detectors.cpp

```diff
@@ -693,7 +693,7 @@ void FacialLandmarksDetection::enqueue(const cv::Mat &face) {
         return;
     }
     if (!request) {
-        request = net.CreateInferRequestPtr();
+        request = std::make_shared<InferenceEngine::InferRequest>(net.CreateInferRequest());
```

Comment:
BTW, do you need shared_ptr? InferRequest itself is a wrapper for shared_ptr<InternalInferRequestInterface>

Reply:
ok, I see you are going to do this separately.